lib/safira/accounts/accounts.ex

```diff
@@ -44,8 +44,17 @@ defmodule Safira.Accounts do
     User.changeset(user, %{})
   end
 
+  def list_leaderboard do
+    Repo.all(from a in Attendee, 
+      where: not (is_nil a.user_id) and not a.volunteer)
+    |> Repo.preload(:badges)
+    |> Enum.map(fn x -> Map.put(x, :badges_count, length(x.badges)) end)
```

Comment:
Why do you you need this ? isn't the badges_count a virtual field auto calculated ? also isn't ``` Enum.sort_by  & &1.badges_count ' ``` a better way of doing the sort ?

Reply:
No, but does it make sense to be? We are gonna have to preload badges all the time. As of this moment we only need this field here

Reply:
It may be but by default it's `<=/2`, but we could change it